core/station/impl/src/core/init.rs

```diff
@@ -164,7 +164,28 @@ lazy_static! {
         (
             Allow::user_groups(vec![*ADMIN_GROUP_ID]),
             Resource::ExternalCanister(ExternalCanisterResourceAction::Fund(ExternalCanisterId::Any)),
-        )
+        ),
+        // assets
+        (
+            Allow::user_groups(vec![*ADMIN_GROUP_ID]),
+            Resource::Asset(ResourceAction::Create),
+        ),
+        (
+            Allow::user_groups(vec![*ADMIN_GROUP_ID]),
+            Resource::Asset(ResourceAction::List),
+        ),
+        (
+            Allow::user_groups(vec![*ADMIN_GROUP_ID]),
+            Resource::Asset(ResourceAction::Read(ResourceId::Any)),
```

Comment:
in the migration we are adding this as a resource for all authenticated users, should we do the same here?

Reply:
good catch